pkgs/by-name/q5/q5go/package.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, libsForQt5
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
```

`rec` is discouraged now, please use

```suggestion
stdenv.mkDerivation (finalAttrs: {
```